executor/executor_test.go

```diff
@@ -215,6 +215,7 @@ func TestShow(t *testing.T) {
 		"RESTRICTED_REPLICA_WRITER_ADMIN Server Admin ",
 	))
 	require.Len(t, tk.MustQuery("show table status").Rows(), 1)
+	tk.MustQuery("show session_states").Check(testkit.Rows())
```

Comment:
Maybe just put it in `TestShowSessionStates`? Why distribute tests all around?

Reply:
`simple.go` covers `executeSetSessionStates` while `show.go` covers `fetchShowSessionStates`.
But I agree that putting them together makes code readable. I'll do it.